lib/appsignal/plug.ex

```diff
@@ -150,7 +150,8 @@ if Appsignal.plug?() do
       %{
         "method" => method,
         "path" => path,
-        "request_id" => request_id
+        "request_id" => request_id,
+        "status" => status
```

Comment:
Is `status` descriptive enough in the context of an AppSignal transaction? (for when it can be jobs etc)
Would `status_code` or `http_status`/`http_status_code` make more sense in the AppSignal.com UI?

As originally commented here: https://github.com/appsignal/appsignal-elixir/pull/452#discussion_r251375553

Reply:
As discussed: going with `http_status_code`

Reply:
Fixed!